azurerm/resource_arm_mssql_elasticpool.go

```diff
@@ -176,7 +176,8 @@ func resourceArmMsSqlElasticPool() *schema.Resource {
 
 			"zone_redundant": {
 				Type:     schema.TypeBool,
-				Computed: true,
+				Optional: true,
+				Default:  false,
```

Comment:
Could we remove the default here and let the API decide by conditionally sending the value?